01_fib/joel00wood/main_test.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"bytes"
+	"io"
+	"log"
+	"os"
+	"sync"
+	"testing"
+)
+
+// enter borrowed code
+// https://gist.github.com/hauxe/e935a7f9012bf2649710cf75af323dbf
+// stdOut capture helper
+func captureOutput(f func()) string {
```

Comment:
I didn't mention this before, but it makes me really happy to see someone avoiding the global `out := os.Stdout` line and doing this "properly"

Given you're using it in all your labs, perhaps this is a good opportunity to explore local packages?